.travis.yml

```diff
@@ -63,10 +63,8 @@ jobs:
 
 
 install:
-  - pip install tox-travis setuptools --upgrade
+  - pip install tox-travis "setuptools<60" --upgrade
```

Comment:
Can you please add a comment to the code why this was needed? So that when we'll need to change it in the future, we'd know what to look out for.

Reply:
Added ✔️